applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -16,6 +16,59 @@ config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
 	  If the option is disabled, the accelerometer trigger handler must manually be enabled to
 	  detect activity.
 
+config EXTERNAL_SENSORS_BME680_BSEC
+	bool "Use Bosch BME680 library"
+	depends on !BME680
+	help
+	  Enable the use of Bosch BSEC library. The library itself is not a part of
+	  NCS and must be downloaded from Bosch Sensortec and placed in the path referenced to by
+	  the EXTERNAL_SENSORS_BME680_BSEC_PATH. This configuration depends on the BME680 zephyr
+	  driver being disabled.
+
+if EXTERNAL_SENSORS_BME680_BSEC
+
+choice EXTERNAL_SENSORS_BSEC_SAMPLE_MODE
+	prompt "Bosch BSEC sample mode"
+	default EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_LOW_POWER
+	help
+	  Configuration that sets how often sensor data is sampled from the BSEC library.
+	  Each mode corresponds an internal preset that decides how often data is sampled from the
+	  BME680.
+
+config EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_ULTRA_LOW_POWER
+	bool "BSEC low ultra power mode"
+	help
+	  Sample data from BSEC every 0.0033333 Hz (300 second interval).
+
+config EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_LOW_POWER
+	bool "BSEC low power mode"
+	help
+	  Sample data from BSEC every 0.33333 Hz (3 second interval).
+
+config CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_CONTINUOUS
+	bool "BSEC continuous mode"
+	help
+	  Sample data from BSEC every 1 Hz (1 second interval).
```

Comment:
**problem:** I still disagree that these configs should be called "sample rate", because they are **boolean** switches, and do not actually allow to configure the sampling rate, but merely select the sensor operating mode.

Reply:
They are not. The are named `_SAMPLE_MODE_` now. The help text simply documents what the mode actually corresponds to in terms of sample rate.